app/models/variant.rb

```diff
@@ -8,5 +8,6 @@ module FlexCommerce
   #
   #
   class Variant < FlexCommerceApi::ApiBase
+    has_one :product
```

Comment:
When you think about it from a JSON point of view - you either have 1 or many - no such thing as has_and_belongs_to_many like in AR as there are no join tables etc..

The belongs_to simply means that this resource is really contained by another, therefore its URL will be expected to define that.  So, unless you really want to do this - always use has_one.

Hope that clarifies it